### PR TITLE
Null Pointer fix for issue #3497

### PR DESCRIPTION
**Pull request by BaronGreenback**
Refreshing the page causes a null pointer as virtualpath is null in GetResourcePath.

### CODE REVIEW

**Comment by crobibero** [comment]
With this change what is the new response?

File: Emby.Server.Implementations/ResourceFileManager.cs [hunk 1]
```diff
@@ -22,7 +22,12 @@ public ResourceFileManager(ILogger<ResourceFileManager> logger, IFileSystem file
 
         public string GetResourcePath(string basePath, string virtualPath)
         {
-            var fullPath = Path.Combine(basePath, virtualPath.Replace('/', Path.DirectorySeparatorChar));
+            var fullPath = basePath;
+
+            if (string.IsNullOrEmpty(virtualPath))
```
Comment (cvium) [inline Emby.Server.Implementations/ResourceFileManager.cs:27]: Missing a !

Comment (BaronGreenback) [reply Emby.Server.Implementations/ResourceFileManager.cs:27]: Yep - was just about to update.

**Comment by BaronGreenback** [comment]
It doesn't error any more.

It doesn't seem to have changed anything else.

I got it on a refresh of the now playing PlayTo screen.

**Comment by BaronGreenback** [comment]
Pulling PR as it has other implications.

**Pull request closed without merging**